amr-wind/diffusion/incflo_diffusion.cpp

```diff
@@ -278,6 +283,164 @@ void wall_model_bc(
     } // level loop
 }
 
+void wall_model_bc_moeng(
+    amr_wind::Field& velocity,
+    const amrex::Real utau,
+    const amr_wind::FieldState fstate,
+    const amrex::FArrayBox& inst_plane_vel,
+    const amrex::Array<amrex::Real, AMREX_SPACEDIM>& velmean,
+    const amrex::Real mean_wind)
+{
+    BL_PROFILE("amr-wind::diffusion::wall_model_bc")
+    auto& repo = velocity.repo();
+    auto& density = repo.get_field("density", fstate);
+    auto& viscosity = repo.get_field("velocity_mueff");
+    const int nlevels = repo.num_active_levels();
+
+    amrex::Orientation zlo(amrex::Direction::z, amrex::Orientation::low);
+    amrex::Orientation zhi(amrex::Direction::z, amrex::Orientation::high);
+
+    auto m_store_xy_vel_arr = inst_plane_vel.array();
```

Comment:
```suggestion
    auto store_xy_vel_arr = inst_plane_vel.array();
```

`m_` prefix reserved for class/struct instance members.